.doc_gen/metadata/textract_metadata.yaml

```diff
@@ -52,5 +69,50 @@ textract_StartDocumentAnalysis:
             - description:
               snippet_tags:
                 - textract.java2._start_doc_analysis.main
+    Python:
+      versions:
+        - sdk_version: 3
+          github: python/example_code/textract
+          excerpts:
+            - description:
+              snippet_tags:
+                - python.example_code.textract.TextractWrapper
+                - python.example_code.textract.StartDocumentAnalysis
+  services:
+   textract: {StartDocumentAnalysis}
+textract_StartDocumentTextDetection:
+  title: Start asynchronous detection of text in a document using &TEXTRACT; and an &AWS; SDK

```

Comment:
I'd shorten the title and move the example-specific info the example description.

`title: Start asynchronous detection of text using &TEXTRACT; and an &AWS; SDK`

Reply:
fixed